### PR TITLE
Further multilingual support: Brands, Collections

### PR DESCRIPTION
**Pull request by UladBohdan**
Also:
* Slug regexp validation for all models

### CODE REVIEW

**Comment by UladBohdan** [comment]
@Drapegnik I've added a new exception into `.eslintrc`, please verify we're fine with that.

**Comment by UladBohdan** [comment]
[slug validation for Articles and Collections](https://trello.com/c/PajScOdb/101-slug-validation-for-articles-and-collections)

**Comment by UladBohdan** [comment]
[consider Brand creation and model Enhancements](https://trello.com/c/pliyr6sp/103-consider-brand-creation-and-model-enhancements)

**Comment by UladBohdan** [comment]
@Drapegnik please review it

File: src/api/article/collection/middlewares.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+export const parseLocalized = (req, res, next) => {
```
Comment (Drapegnik) [inline src/api/article/collection/middlewares.js]: I think it's redundant, cause we already have [`body-parser`](https://github.com/babajka/babajka-backend/blob/master/src/server.js#L19)

File: src/api/article/collection/tests.js [hunk 2]
```diff
@@ -121,7 +121,7 @@ describe('Collections API', () => {
         .set('Cookie', sessionCookie)
         .send({
           slug: 'collection-6',
-          name: 'New Collection',
+          name: `{ "en": "New Collection" }`,
```
Comment (Drapegnik) [inline src/api/article/collection/tests.js]: try with object instead string here

**Pull request merged**